rsbuild.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "@rsbuild/core";
+import { pluginReact } from "@rsbuild/plugin-react";
+
+export default defineConfig({
+    html: {
+        template: "./index.html",
```

Comment:
The reason having it here is because the project used to be a vite one which has the html file as an entry. I think it would also work if we have a `index.ts` file that calls the `createReactRoot().render()`